content/docs/details/techdocs/index.md

```diff
@@ -72,21 +72,27 @@ NB: Some styles will always be overridden regardless of the mkdocs-material plug
 
 ## Testing and previewing your documentation
 
-You can generate / serve your docs locally to view what they would look like when they are deployed to Roadie.
+To test documentation changes we suggest you register an org or team scoped entity in the catalog like so: 
```

Comment:
This wont work for testing in a CI workflow. Can we split out an automated testing section?

Also there is a getting started tech docs page that you might want to edit also.

Reply:
You are correct